teszt.py

```diff
@@ -0,0 +1,3 @@
+#kérj be a felhasználótól 50-től 67-ig számokat
+for b range(50,68,1).
+  print(B)
```

Comment:
print(b)

írd ki 50-től 67-ig a számokat*

Reply:
Szép múnka

Reply:
valamit nem vetél észre

Reply:
mit?

Reply:
for b in range(50,68,1):

mar megvan

Reply:
2

Reply:
ja